llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -8845,9 +8848,13 @@ Instruction *SPIRVToLLVM::transMemFence(BasicBlock *BB, SPIRVWord MemSema,
     Ordering = AtomicOrdering::Acquire;
   else if (MemSema & MemorySemanticsReleaseMask)
     Ordering = AtomicOrdering::Release;
-
-  if (Ordering == AtomicOrdering::NotAtomic)
-    return nullptr;
+  else {
```

Comment:
nit: could you add braces around the if and else-if too?

Reply:
Done.

Incidentally I have just noticed two other places in the same file where we translate SPIR-V memory ordering to LLVM memory ordering: transMemorySemantics and transSPIRVImageAtomicOpFromInst.

Reply:
@trenouf So I think for single statement in if-else. I think we can let developers decide whether {} is needed according to the complexity of the code logic. This could be stated in the coding style doc.

Reply:
I think Jakub is suggesting that we follow the (unstated) LLVM convention that:
1. you don't need braces for a single-line single-statement body
2. for if..else (or a chain of if..else if .. else if .. else), then if you need braces in one body you should put braces on all the other bodies.

The coding style doc only states (1). I should add (2) too.

Reply:
Oh, I see. I didn't realize LLVM has such requirements for (2). Thanks.